controllers/vrg_kubeobjects.go

```diff
@@ -187,7 +187,7 @@ func kubeObjectsCaptureStartConditionallySecondary(
 
 func kubeObjectsCaptureStartConditionallyPrimary(
 	v *VRGInstance, result *ctrl.Result,
-	captureStartGeneration int64, captureStartTimeSince, captureStartInterval time.Duration,
+	_ int64, captureStartTimeSince, captureStartInterval time.Duration,
```

Comment:
This one too.

Reply:
This function must have a signature:
captureStartConditionally func(*VRGInstance, *ctrl.Result, int64, time.Duration, time.Duration, func())